__TESTS__/TestUtils/expectESMToMatchDefault.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @description given an ESM package and a default export, this function expects the two to be identical
```

Comment:
should be part of a different pr?

Reply:
I had to write all this code for the tests, all I did was just put it in a different function. 
I believe it belongs here